harness/benchmarks/src/main/kotlin/godot/benchmark/Simple.kt

```diff
@@ -47,4 +53,21 @@ class Simple : Object() {
         return s
     }
 
+    @RegisterFunction
+    fun benchmarkICall(): Node {
+        val node = Node()
+        node.getInstanceId()
+        return node
+    }
+
```

Comment:
Annotation missing. But not really important as the entry generation is not yet added to the benchmarks.
Maybe we should add this next?

Reply:
Yep, purpose here, was not to convert benchmarks with annotations, only optimizing.